lib/mix/lib/mix/dep/converger.ex

```diff
@@ -48,6 +48,9 @@ defmodule Mix.Dep.Converger do
   """
   def converge(acc, lock, opts, callback) do
     {deps, acc, lock} = all(acc, lock, opts, callback)
+    remote = Mix.RemoteConverger.get
+    # TODO: Remove function_exported? check before releasing 1.4
```

Comment:
Would it make sense to open an issue for every TODO currently present in the source code?